weather@mockturtl/files/weather@mockturtl/3.4/applet.js

```diff
@@ -0,0 +1,2345 @@
+"use strict";
```

Comment:
If you're going to transpile with Babel, please include the original source file somewhere, along with the build scripts used to generate it.

Reply:
i will do that, don't know why I did not include them